functions/knative/java/hello-world/README.md

```diff
@@ -0,0 +1,100 @@
+# Sample Function Java
+
+## Prerequisites
+
+### OpenFunction
+
+You can refer to the [Installation Guide](https://openfunction.dev/docs/getting-started/installation/) to setup OpenFunction.
+
+## Run it locally
+
+Build the function locally
+
+  ```sh
+  pack build func-http-java --path ../ --builder openfunction/builder-java:v2-18 --env FUNC_NAME="dev.openfunction.samples.HttpFunctionImpl"  --env FUNC_CLEAR_SOURCE=true
+  ```
+
+Run the function
+
+  ```sh
+  docker run --rm --env="FUNC_CONTEXT={\"name\":\"HelloWorld\",\"version\":\"v1.0.0\",\"port\":\"8080\",\"runtime\":\"Knative\"}" --env="CONTEXT_MODE=self-host" --name func-http-java -p 8080:8080 func-http-java
+  ```
+
+Send a request
+
+  ```sh
+  curl http://localhost:8080/
+  # Hello World
+  ```
+
+## Deployment
+
+1. Create secret
+
+Follow [this guide](../../../../Prerequisites.md#registry-credential) to create a registry credential.
```

Comment:
Use the prereq guide in : https://openfunction.dev/docs/getting-started/quickstarts/prerequisites/